api/envoy/config/route/v3/route_components.proto

```diff
@@ -655,6 +655,12 @@ message CorsPolicy {
   // Envoy will lookup the runtime key to get the percentage of requests for which it will evaluate
   // and track the request's *Origin* to determine if it's valid but will not enforce any policies.
   core.v3.RuntimeFractionalPercent shadow_enabled = 10;
+
+  // Specify whether allow requests whose target server's IP address is more private than that from
+  // which the request initiator was fetched.
+  //
+  // More details refer to https://developer.chrome.com/blog/private-network-access-preflight.
+  google.protobuf.BoolValue allow_private_network_access = 12;
```

Comment:
What is the default value?

Reply:
`google.protobuf.BoolValue` is false as default, see https://developers.google.com/protocol-buffers/docs/proto#optional